CMakeLists.txt

```diff
@@ -37,6 +37,7 @@ if(CMAKE_PROJECT_NAME STREQUAL PROJECT_NAME)
     SET(KEEP_SYMBOLS "/DEBUG:FULL")
     SET(STRIP_SYMBOLS "/DEBUG:NONE")
     SET(PROFILE "/PROFILE")
+    SET(ADDRESS_SANITIZE "/fsanitize=address /Oy-")
```

Comment:
what does this do?

Reply:
these are the equivalent flags using the MSVC compiler to `-fsanitize=address -fno-omit-frame-pointer`